RealmTasks Android/app/src/main/java/io/realm/realmtasks/TaskListActivity.java

```diff
@@ -156,21 +156,21 @@ public boolean onOptionsItemSelected(MenuItem item) {
         public void onMoved(RecyclerView recyclerView, ItemViewHolder from, ItemViewHolder to) {
             final int fromPosition = from.getAdapterPosition();
             final int toPosition = to.getAdapterPosition();
+            // TODO need to suppress executing listener after https://github.com/realm/realm-java/issues/4225 is introduced.
             adapter.onItemMoved(fromPosition, toPosition);
-            adapter.notifyItemMoved(fromPosition, toPosition);
         }
 
         @Override
         public void onCompleted(ItemViewHolder viewHolder) {
+            // TODO need to suppress executing listener
             adapter.onItemCompleted(viewHolder.getAdapterPosition());
-            adapter.notifyDataSetChanged();
```

Comment:
if this is removed, then the line gets stuck in green if the list had no items in it.